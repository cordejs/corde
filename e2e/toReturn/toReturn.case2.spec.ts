import { runCLI } from "../cliRunner";
import Utils from "../testUtils";

it("tests should not be sucessfull", async () => {
  const command = Utils.buildCommandWithConfigPath("toReturn", "bot_case2.spec.ts");
  const [mockProcess, stdout] = await runCLI(command);
  expect(mockProcess).toBeCalledWith(1);
  expect(stdout).toMatchSnapshot();
});
