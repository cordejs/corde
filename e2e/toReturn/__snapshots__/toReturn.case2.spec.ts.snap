// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests should not be sucessfull 1`] = `
"node test --config ./e2e/corde.config.ts --files ./e2e/toReturn/__cordeTest__/bot_case2.spec.ts
opa

 FAIL   e2e\\\\toReturn\\\\__cordeTest__\\\\bot_case2.spec.ts   100ms
   x Hello command should return... hello!!   100ms
    
    expected: '...'
    received: 'Hello!!'

    at toReturn e2e\\\\toReturn\\\\__cordeTest__\\\\bot_case2.spec.ts:6:35
    at _internalTest (lib\\\\src\\\\closures\\\\test.js:29:13)
    at lib\\\\src\\\\closures\\\\test.js:52:65

Test Files: 1 failed. 1 total
Tests:      1 failed. 1 total
Time:       100ms
"
`;
